config/config.go

```diff
@@ -54,10 +54,11 @@ type Config struct {
 	MultipleAnaxInstances            bool      // multiple anax instances running on the same machine
 	DefaultServiceRetryCount         int       // the default service retry count if retries are not specified by the policy file. The default value is 2.
 	DefaultServiceRetryDuration      uint64    // the default retry duration in seconds. The next retry cycle occurs after the duration. The default value is 600
-	ServiceConfigStateCheckIntervalS int       // the service configuration state check interval. The default is 30 seconds.
+	ServiceConfigStateCheckIntervalS int       // deprecated. The service configuration state check interval. The default is 30 seconds.
 	DefaultNodePolicyFile            string    // the default node policy file name.
+	NodeCheckIntervalS               int       // the node check interval. The default is 15 seconds.
 	NodePolicyCheckIntervalS         int       // the node policy check interval. The default is 15 seconds.
-	NodeUserInputCheckIntervalS      int       // the node user input check interval. The default is 15 seconds.
+	NodeUserInputCheckIntervalS      int       // deprecated. The node user input check interval. The default is 15 seconds.
```

Comment:
It doesnt look like there is any code which references this field, so I think you should delete it.

Reply:
Done